wurst/systems/modes/GameMode.wurst

```diff
@@ -426,7 +425,7 @@ init
         mode.message("|c00ae4946Slow mode|r has been enabled. Less items spawn at once.")
 
     new GameMode("lower-food-limit", "lfl", "Lower spawn rate for animals on the map") (mode, args) ->
-        udg_ANIMAL_MAX = 60
+        udg_ANIMAL_MAX = 50
```

Comment:
Should this reference the variable you created in `gameConfig`?

Reply:
Yes

Reply:
Changed